components/network/quagga/OpenIndiana/README.OpenIndiana

```diff
@@ -0,0 +1,461 @@
+#
```

Comment:
Please, move files to quagga/files or keep as quagga/Solaris, we have no component/OpenIndiana precedents.